### PR TITLE
(+semver: fix) Update protoc usage to work with macOS and newer golang

### PR DESCRIPTION
**Pull request by annymsMthd**
Signed-off-by: Joshua Benjamin <joshua.benjamin@iherb.com>

### CODE REVIEW

**Comment by annymsMthd** [comment]
Not sure if the update to go 1.16 will mess up anything on your end. Let me know and I can just pull that out of this PR. This should fix kafmesh-gen on mac. So far I have it tested on bigsur. This also should let you use the latest version of protoc and the protoc-gen-go

**Comment by annymsMthd** [comment]
Also I updated sarama. I'm going to be running this against kafka sometime this weekend and I can report back if I had any issues. If there is I can just remove those updates also.

**Comment by annymsMthd** [comment]
#32 linking issue

**Comment by annymsMthd** [comment]
Looks like this isn't exporting the grpc client/server with the protobuf generation. I need to take a look at that.

**Comment by annymsMthd** [comment]
fixed

**Comment by annymsMthd** [comment]
Got more of this up and running. Sarama 1.28 is breaking partitioning for joins and will put pods into a CrashLoopBackoff. Rolling back to 1.27 fixes the issue.. Looks like it still needs to be pinned there for now

File: go.mod [hunk 1]
```diff
@@ -1,39 +1,66 @@
...
-go 1.13
+go 1.16
 
 require (
-	github.com/99designs/gqlgen v0.11.3
-	github.com/Shopify/sarama v1.26.1
-	github.com/avast/retry-go v2.5.0+incompatible // indirect
+	github.com/99designs/gqlgen v0.13.0
+	github.com/Microsoft/go-winio v0.5.0 // indirect
+	github.com/Shopify/sarama v1.28.0
```
Comment (cuevaskoch) [inline go.mod:8]: Can you update this to pin to 1.27 per [your comment](https://github.com/syncromatics/kafmesh/pull/33#issuecomment-834735534)?

**Review by cuevaskoch — CHANGES_REQUESTED** [review]

File: go.mod (hunk 1 above)
Comment (cuevaskoch) [reply go.mod]: Thanks!

**Review by cuevaskoch — APPROVED** [review]

**Review by thzinc — APPROVED** [review]

**Pull request merged**